# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/mepo/0001-zig-0.13-use-std.Build.path.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

From a5fb84751db02c4f913199fd28bce3751a9bc485 Mon Sep 17 00:00:00 2001
From: "lauren n. liberda" <lauren@selfisekai.rocks>
Date: Sat, 8 Jun 2024 02:38:48 +0200
Subject: [PATCH] zig 0.13: use std.Build.path

---
 build.zig | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/build.zig b/build.zig
index a61082e..923eca5 100644
--- a/build.zig
+++ b/build.zig
@@ -11,7 +11,7 @@ fn setDependencies(step: *std.Build.Step.Compile) void {
 
 pub fn build(b: *std.Build) void {
     b.installDirectory(.{
-        .source_dir = .{ .path = "scripts" },
+        .source_dir = b.path("scripts"),
         .install_dir = .{ .bin = {} },
         .install_subdir = "",
     });
@@ -24,7 +24,7 @@ pub fn build(b: *std.Build) void {
     const optimize = b.standardOptimizeOption(.{});
     const exe = b.addExecutable(.{
         .name = "mepo",
-        .root_source_file = .{ .path = "src/main.zig" },
+        .root_source_file = b.path("src/main.zig"),
         .target = target,
         .optimize = optimize,
     });
@@ -42,7 +42,7 @@ pub fn build(b: *std.Build) void {
 
     // Setup test
     const tests = b.addTest(.{
-        .root_source_file = .{ .path = "./src/test.zig" },
+        .root_source_file = b.path("./src/test.zig"),
         .target = target,
         .optimize = optimize,
     });
